src/components/Edit.js

```diff
@@ -101,6 +103,13 @@ class Edit extends Component {
       })
   }
 
+  onAliasChange (alias) {
+    this.setState({ alias })
+    this._document.peers.setPeerAlias(alias)
```

Comment:
@pgte would be good to set the alias our local alias on the peers in the constructor as well, or we have to go click set.

Reply:
It's being done here: https://github.com/ipfs-shipyard/peerpad/pull/116/files#diff-96eeb67cd2b0e6e2fe9df14e4c8036f3R208